arduino/smartcar/smartcar.ino

```diff
@@ -156,3 +169,7 @@ void rotateOnSpot(int targetDegrees) {
 int measureDistance() {
       return front.getDistance();
 }
+
+int travelledDistance() {
+      return leftOdometer.getDistance();
```

Comment:
Why only from the left odometer and not from both? The `DistanceCar` class (and therefore the `SmartCar` one too) has a function just for that.